crates/client/db/src/lib.rs

```diff
@@ -19,7 +19,7 @@ pub use mapping_db::MappingCommitment;
 use starknet_api::hash::StarkHash;
 mod da_db;
 mod db_opening_utils;
-mod messaging_db;
+pub mod messaging_db;
```

Comment:
We don't want any of the db `mod` to be public. They should only be accessible through our `Backend` struct

Reply:
I would reexport `LastSyncedEventBlock` struct in `lib.rs`